migrate/20251031_add_stack_to_lb_vm_port.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  up do
+    create_enum(:lb_vm_port_stack, %w[ipv4 ipv6])
+    alter_table(:load_balancer_vm_port) do
+      add_column :stack, :lb_vm_port_stack, null: false, default: "ipv4"
```

Comment:
I think this is trying to do too much at once. Add the column allowing NULL in one migration. Modify the code to start setting the value for the column. After that, backfill the previously existing fields that are still NULL. Once that is complete, then add the NOT NULL constraint in a separate migration. I don't think you should have a DEFAULT on the column.